mysite/contributors.md

```diff
@@ -0,0 +1,24 @@
+# **OUR TOP CONTRIBUTORS !!!!**
+![Thank you Image](https://pvsmt99345.i.lithium.com/t5/image/serverpage/image-id/11515i5FE675D18BD5D6D4/image-size/large?v=1.0&px=999)
+
+# Top contributors!!  :+1
```

Comment:
Hey @shraddhavp I am still unable to see the emoji here, probably you missed the `:` here i.e ` :+1: ` Please add the colon after `1`.